src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,56 @@
+package duke;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import duke.command.Command;
+import duke.task.TaskList;
+import javafx.application.Platform;
+
+/**
+ * Personal Assistant that keeps track of a user's list of tasks.
+ */
+public class Duke {
+    private static final Path storageFilePath = Paths.get(".", "data", "test.txt");
+    private Storage storage;
+    private TaskList taskList;
+
+    /**
+     * Constructor to create a Duke object.
+     * <p>
+     * Duke consists of a <code>TaskList</code>, and <code>Storage</code> which
+     * is responsible for user interactions, keeping track of user's tasks, and writing tasks
+     * to the user's local storage respectively.
+     */
+    public Duke() {
+        this.storage = new Storage(Duke.storageFilePath);
+        try {
+            this.taskList = new TaskList(this.storage.getAllTasks());
+        } catch (DukeException e) {
+            this.taskList = new TaskList();
+        }
+    }
+
+    /**
+     * Gets Duke's response to the user input.
+     *
+     * @param input the input by the user via the GUI.
+     * @return the response given by Duke to the user input.
+     */
+    public String getResponse(String input) {
+        try {
+            Command c = Parser.parse(input);
```

Comment:
Perhaps you could rename the variable `c` to `command` instead? to make line 43 and 47 clearer